### DISCUSSION TITLE
Ctrl+F doesn't work when parts of the DOM are hidden by the "read more" button

### DISCUSSION THREAD

**Discussion by Tanja-4732** [community/community#6515] [Other Feature Feedback, Questions, & Ideas] [answered]
The "read more" button inserts new DOM nodes, meaning they didn't exist before, breaking Ctrl+F or F3 search.

This is a feature-regression of the new design, and should be treated as a bug/regression IMHO.

At least give the users an option to "read more" on all entries by default, essentially getting rid of that button for that specific user.

What's your opinion on this?

**Comment by MylesBorins** [comment, marked as answer]
@Bernd-L if you use the search at the top of the page we return results with all releases expanded by default. You can append `?expanded=true` to the release page if you want all releases expanded by default. While this is a "regression" in some capacity (losing the ability to quickly search the first page of releases) we do believe the overall experience is improved and our built in search is far more accurate allowing folks to filter by keyword, tag, or date.

**Reply by Tanja-4732** [reply]
Thanks for your answer!
Could there be a toggle in GH user settings (or somewhere else) to always expand by default, or are we going to need a browser extension to automatically append `?expanded=true` on every applicable URL?

**Reply by MylesBorins** [reply]
we are currently discussing if this is something that we want to expose as an explicit feature or not. At the very least we did want to ensure that the capability was there so browser extensions could be used if necessary

**Reply by panva** [reply]
> we do believe the overall experience is improved and our built in search is far more accurate allowing folks to filter by keyword, tag, or date.

@MylesBorins I agree the built in search is very powerful, that being said, when looking for "something i know is there" that search cannot compete with hitting CMD+F and just hammering keywords until what's needed is found.

**Reply by jsunsawyer** [reply]
@MylesBorins What improvement does collapsing these provide? Is it simply to show less content, so the user can scan through more releases?

Is the intent to make it easier for users to browse several, old releases?

I'd think that people more often view releases to find a recent breaking change, fix, or feature, and generally only care about a few recent releases. There's no need to search all releases, and like @panva mentioned, it can be easier to simply find content in the page.

That said, if you are keeping these collapsed, but making a change to show the latest release as expanded, this could be confusing to users, as they may expect subsequent releases to also show expanded results that can be found on page with CMD+F.
